kornia/color/luv.py

```diff
@@ -0,0 +1,191 @@
+from typing import Tuple
+import torch
+import torch.nn as nn
+
+from .xyz import rgb_to_xyz, xyz_to_rgb
+
+"""
+The RGB to Luv color transformations were translated from scikit image's rgb2luv and luv2rgb
+
+https://github.com/scikit-image/scikit-image/blob/a48bf6774718c64dade4548153ae16065b595ca9/skimage/color/colorconv.py
+
+"""
+
+
+class RgbToLuv(nn.Module):
+    r"""Converts an image from RGB to Luv
+
+    The image data is assumed to be in the range of :math:`[0, 1]`. Luv
+    color is computed using the D65 illuminant and Observer 2.
+
+    args:
+        image (torch.Tensor): RGB image to be converted to Luv.
+    returns:
+        torch.Tensor: Luv version of the image.
+    shape:
+        - image: :math:`(*, 3, H, W)`
+        - output: :math:`(*, 3, H, W)`
+    Examples:
+        >>> input = torch.rand(2, 3, 4, 5)
+        >>> luv = kornia.color.RgbToLuv()
+        >>> output = luv(input)  # 2x3x4x5
+    Reference:
+        [1] https://docs.opencv.org/4.0.1/de/d25/imgproc_color_conversions.html
+
+        [2] https://www.easyrgb.com/en/math.php
+
+        [3] http://www.poynton.com/ColorFAQ.html
+    """
+
+    def __init__(self) -> None:
+
+        super(RgbToLuv, self).__init__()
+
+    def forward(self, image: torch.Tensor) -> torch.Tensor:  # type: ignore
+
+        return rgb_to_luv(image)
+
+
+class LuvToRgb(nn.Module):
+    r"""Converts an image from Luv to RGB
+
+    args:
+        image (torch.Tensor): Luv image to be converted to RGB.
+    returns:
+        torch.Tensor: RGB version of the image.
+    shape:
+        - image: :math:`(*, 3, H, W)`
+        - output: :math:`(*, 3, H, W)`
+    Examples:
+        >>> input = torch.rand(2, 3, 4, 5)
+        >>> rgb = kornia.color.LuvToRgb()
+        >>> output = rgb(input)  # 2x3x4x5
+    References:
+        [1] https://docs.opencv.org/4.0.1/de/d25/imgproc_color_conversions.html
+
+        [2] https://www.easyrgb.com/en/math.php
+
+        [3] http://www.poynton.com/ColorFAQ.html
+    """
+
+    def __init__(self) -> None:
+
+        super(LuvToRgb, self).__init__()
+
+    def forward(self, image: torch.Tensor) -> torch.Tensor:  # type: ignore
+
+        return luv_to_rgb(image)
+
+
+def rgb_to_luv(image: torch.Tensor) -> torch.Tensor:
+    r"""Converts a RGB image to Luv.
+
+    See :class:`~kornia.color.RgbToLuv` for details.
+
+    Args:
+        image (torch.Tensor): RGB image
+    Returns:
+        torch.Tensor : Luv image
+    """
+
+    if not torch.is_tensor(image):
+        raise TypeError("Input type is not a torch.Tensor. Got {}".format(
+            type(image)))
+
+    if len(image.shape) < 3 or image.shape[-3] != 3:
+        raise ValueError("Input size must have a shape of (*, 3, H, W). Got {}"
+                         .format(image.shape))
+
+    # Convert from Linear RGB to sRGB
```

Comment:
not for this PR, but maybe we could have an intermediate  `rgb_to_srgb` ?

Reply:
Yeah it could be useful since the opencv lab color uses the linear RGB to srgb conversion.